aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -164,8 +185,8 @@ Resources:
               - 'dynamodb:PutItem'
               - 'dynamodb:Scan'
             Resource:
-              - !Sub "arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/<%=CDO.dcdo_table_name%>"
-              - !Sub "arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/<%=CDO.gatekeeper_table_name%>"
+              - !Sub "arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/${DCDO_Table_Name}"
+              - !Sub "arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/${Gatekeeper_Table_Name}"
```

Comment:
This is a good example of one of the changes in this PR. I'm hoisting references to `CDO` properties out of the templates and making them stack parameters. I have not eliminated the calls to `CDO`, but I've moved them up a layer.